userguide/tasks.rst

```diff
@@ -1127,8 +1127,8 @@ method is selected.
    | Stream Compression        | SSH       | drop-down menu | Select a compression algorithm to reduce the size of the data being replicated. Only appears when *SSH* is      |
    |                           |           |                | chosen for :guilabel:`Transport`.                                                                               |
    +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Limit (Ex. 500 KiB,       | SSH       | integer        | Limit replication speed to this number of bytes per second. Zero means no limit. Units like :literal:`k`,       |
-   | 500M, 2 TB)               |           |                | :literal:`KiB`, and :literal:`M` can be used. Numbers without unit letters are interpreted as bytes.            |
+   | Limit (Ex. 500 KiB,       | SSH       | integer        | Limit replication speed to this number of bytes per second. Zero means no limit. This feild is                  |
+   | 500M, 2 TB)               |           |                | :ref:`human readable <Human Readable Fields>`. Numbers without unit letters are interpreted as bytes.           |
```

Comment:
The sentence about numbers without units and the example should also be removed.